src/routes/users/updateUser.test.ts

```diff
@@ -0,0 +1,64 @@
+import { FastifyInstance, RouteOptions } from "fastify";
+import createServer from "../../server";
+import updateUser from "./updateUser";
+import UserModel from "../../models/user";
+
+describe("PUT /user", () => {
+  let server: FastifyInstance;
+  beforeAll(() => {
+    server = createServer({ authRoute: [updateUser] as RouteOptions[] });
+  });
+
+  it("should update a user correctly", async () => {
```

Comment:
**Should** always refers to something that may or may not happen, so we prefer to use the third person here since we have IT.

updates a user correctly